src/python/turicreate/toolkits/object_detector/_tf_image_augmenter.py

```diff
@@ -22,351 +22,351 @@
 
 tf.disable_v2_behavior()
 
-def get_augmented_data(images, annotations, output_height, output_width, resize_only):
+_DEFAULT_AUG_PARAMS = {
+  'max_hue_adjust' : 0.05,
+  'max_brightness' : 0.05,
+  'max_contrast' : 1.25,
+  'max_saturation' : 1.25,
+  'flip_pr' : True,
+  'skip_probability' : 0.1,
+  'min_aspect_ratio' : 0.8,
+  'max_aspect_ratio' : 1.25,
+  'min_area_fraction' : 1.0,
+  'max_area_fraction' : 2.0,
+  'max_attempts' : 50,
+  'output_shape' : (416, 416),
```

Comment:
Many of these appear to be unused. Can we find some consistent way to reason about / collect these constants?

Reply:
Just put in the right place. Doing it now.